azure/table_azure_api_management_backend.go

```diff
@@ -0,0 +1,225 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/apimanagement/mgmt/2020-12-01/apimanagement"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION ////
+
+func tableAzureAPIManagementBackend(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_api_management_backend",
+		Description: "Azure API Management Backend",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"backend_id", "resource_group", "service_name"}),
```

Comment:
Can we consider using just one required column, `id`, as the mandatory parameter, and then parse this `id` to extract `backend_id`, `resource_group`, and `service_name`?